rustls/src/alias.rs

```diff
@@ -0,0 +1,5 @@
+#[cfg(not(feature = "withrcalias"))]
```

Comment:
This should just be an inline module in `lib.rs`.

Reply:
should be resolved: now moved into the inline module